docs/ruleset_functions/check_purchased_chw_hhw.md

```diff
@@ -0,0 +1,75 @@
+
+## check_purchased_chw_hhw
+
+Description: Check if RMR is modeled with purchased chilled water as space cooling source or purchased hot water/steam as space heating source. If any system in RMR uses purchased chilled water, function shall return True for purchased chilled water as space cooling source. Similarly, if any system in RMR uses purchased hot water or steam, function shall return True for purchased hot water/steam as space heating source.
+
+Inputs:  
+- **RMR**: The RMR that needs to be checked.
+
+Returns: 
+- **purchased_chw_hhw_status_dictionary**: A dictionary that saves whether RMR is modeled with purchased chilled water as space cooling source or purchased hot water/steam as space heating source, i.e. {"PURCHASED_COOLING": TRUE, "PURCHASED_HEATING": FALSE}.
+
+Function Call:
+- GET_COMPONENT_BY_ID()
+
+Logic:  
+
+- Check if RMR is not modeled with external fluid source: `if NOT RMR.ASHRAE229.external_fluid_source:`
```

Comment:
Don't we need to return the dictionary in this case as well? Suggesting moving line 19 above line 17.